src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
```

Comment:
If this is such an important section that you have to add a large comment, why don't you move the type sin their own file?

Reply:
It really don't be needed. I removed it, and types have been moved to their own file in the [last commit](https://github.com/marmelab/kuba-integration/commit/52c48e332abaf5397cf6debd1512e2111062ba31)